content/blog/2022-03-08-react-18-upgrade-guide.md

```diff
@@ -248,10 +247,36 @@ In this release, React is dropping support for Internet Explorer, which is [goin
 
 If you need to support Internet Explorer we recommend you stay with React 17.
 
-## Other Changes
+## Deprecations
```

Comment:
We might want to run `yarn generate-ids` before we could push these changes to the main branch

Reply:
Never mention only blogs don't seem to have titles

Reply:
Good catch, just ran it and found a few headings we need